### ISSUE TITLE
[Bug]: playback error not handled cleanly

### ISSUE THREAD

**Issue by mrjvs** [movie-web/movie-web#847]
### What happened?

Any playback error caused a full page crash, no way to retry the stream or choose another source.

### What browsers are you seeing the problem on?

Firefox, Chrome

### Steps to reproduce?

1. Grab random non-video link like Google.com
2. Go to /dev/video
3. Paste link
4. Get forced playback error

**Comment by KaKashi1210** [comment]
hey @mrjvs  how would you like to tackle this issue ??

**Comment by KaKashi1210** [comment]
xD let me rephrase: how would you like to fix this issue :)

**Comment by arrzdev** [comment]
I have been working on this issue and have a proposal for this.
Want to check with someone if what I have is the approach intended going forward so if someone is available please drop discord @.
Also please assign me this issue, thanks.

**Comment by arrzdev** [comment]
@mrjvs could you assign me and hop on a discord call w me when possible?

**Comment by mrjvs** [comment]
> @mrjvs could you assign me and hop on a discord call w me when possible?

I no longer maintain movie-web, I suggest you reach out to someone in the discord server: https://movie-web.github.io/links/discord
